src/transformers/pipelines/__init__.py

```diff
@@ -784,6 +791,18 @@ def pipeline(
         # so the model_config might not define a tokenizer, but it seems to be
         # necessary for the task, so we're force-trying to load it.
         load_tokenizer = True
+    if (
```

Comment:
To solve some issues, but in generally, we should have this block too, just as for tokenizer/feature_extractor above and below.